<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMR_SATIS_FORMU</class>
 <widget class="QDialog" name="DMR_SATIS_FORMU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sabit Kıymetler Satışı</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,10">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="kernel_buttons_widget" native="true"/>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sabit Kıymet Kodu / Adı  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
       <item>
        <widget class="QSearchEdit" name="searchEdit_dmr_kodu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>Arama widget. </string>
         </property>
         <property name="QSearchEdit" stdset="0">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>100</height>
          </rect>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_dmr_ismi">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Faturası</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_faturasi">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_fatura_sec">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../e9_icons.qrc">
         <normaloff>:/e9_icons/E9_ICONS/fatura.png</normaloff>:/e9_icons/E9_ICONS/fatura.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>238</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSearchEdit</class>
   <extends>QWidget</extends>
   <header>QSearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../e9_icons.qrc"/>
 </resources>
 <connections/>
</ui>
